src/frameworks/uikit/ui_color.rs

```diff
@@ -67,6 +67,13 @@ pub const CLASSES: ClassExports = objc_classes! {
     autorelease(env, new)
 }
 
++ (id)colorWithWhite:(CGFloat)w
+             alpha:(CGFloat)a {
```

Comment:
Nitpick: the style touchHLE tries to follow for Objective-C methods is that you align the colons on each line, so:

```objc
 (id)colorWithWhite:(CGFloat)w
              alpha:(CGFloat)a {
```

Reply:
Though since there's only two arguments here, you could make this into a one-liner. Your choice.

Reply:
resolved